### PR TITLE
added astrocloud init

### PR DESCRIPTION
**Pull request by jeremyholtzman**
## Description & motivation

I was following along the guide but I got an error message saying that my directory wasn't as Astronomer project when I tried to run `astrocloud dev start`. The terminal told me to first initialize the Astronomer project (which I did) and then I was able to successfully run the dev start command.

### CODE REVIEW

**Review by christineberger — DISMISSED** [review]
Looks good to me!

File: website/docs/guides/orchestration/airflow-and-dbt-cloud/2-setting-up-airflow-and-dbt-cloud.md [hunk 1]
```diff
@@ -36,9 +36,10 @@ cd airflow-dbt-cloud
 
 ## 4. Start the Docker container
```
Comment (runleonarun) [inline website/docs/guides/orchestration/airflow-and-dbt-cloud/2-setting-up-airflow-and-dbt-cloud.md:37]:
```suggestion
## 4. Start the Docker container

You can initialize an Astronomer project in an empty local directory using a Docker container, and then run your project locally using the `start` command.
```

File: website/docs/guides/orchestration/airflow-and-dbt-cloud/2-setting-up-airflow-and-dbt-cloud.md [hunk 2]
```diff
@@ -36,9 +36,10 @@ cd airflow-dbt-cloud
...
 
-1. Run the following command to spin up the Docker container and start your local Airflow deployment:
+1. Run the following commands to turn your directory into an Astronomer project and spin up the Docker container and start your local Airflow deployment:
```
Comment (runleonarun) [inline website/docs/guides/orchestration/airflow-and-dbt-cloud/2-setting-up-airflow-and-dbt-cloud.md]:
```suggestion
1. Run the following commands to initialize your project and start your local Airflow deployment:
```

**Review by runleonarun — DISMISSED** [review]
Thanks so much @jeremyholtzman for this great addition! I'd just like to pull out some of the overarching content before people get to the steps.

**Review by runleonarun — APPROVED** [review]
LGTM!

**Pull request merged**